libcxx/include/__algorithm/stable_sort.h

```diff
@@ -253,6 +253,12 @@ _LIBCPP_CONSTEXPR_SINCE_CXX26 void __stable_sort(
   if constexpr (__allowed_radix_sort) {
     if (__len <= __buff_size && __len >= static_cast<difference_type>(__radix_sort_min_bound<value_type>()) &&
         __len <= static_cast<difference_type>(__radix_sort_max_bound<value_type>())) {
+      for (auto* p = __buff; p < __buff + __buff_size; ++p) {
```

Comment:
This looks disgusting, but taking `__construct_at` inside radix_sort would degrade the design of the algorithm.
Maybe the better solution exists?